packages/@react-stately/table/src/TableCollection.ts

```diff
@@ -16,10 +16,12 @@ import {TableCollection as ITableCollection} from '@react-types/table';
 import {Key} from 'react';
 
 interface GridCollectionOptions {
-  showSelectionCheckboxes?: boolean
+  showSelectionCheckboxes?: boolean,
+  showDragButtons?: boolean
 }
 
 const ROW_HEADER_COLUMN_KEY = 'row-header-column-' + Math.random().toString(36).slice(2);
+const ROW_HEADER_COLUMN_KEY_DRAG = 'row-header-column-' + Math.random().toString(36).slice(2);
```

Comment:
this and ROW_HEADER_COLUMN_KEY theoretically could be the same, we should check for a conflict and generate a new one if they do